orm/sequence_test.go

```diff
@@ -1,52 +1,41 @@
 package orm
 
 import (
-	"bytes"
-	"fmt"
 	"testing"
 
-	"github.com/iov-one/weave"
 	"github.com/iov-one/weave/store"
-	"github.com/stretchr/testify/assert"
 )
 
 func TestSequence(t *testing.T) {
 	db := store.MemStore()
-
-	cases := []struct {
-		bucket, name string
-		init         int64
-		increments   int64
-	}{
-		0: {"a", "bc", 0, 22},
-		1: {"ab", "c", 0, 11},
-		2: {"a", "bc", 22, 18},
-		3: {"", "abc", 0, 77},
-		4: {"ab", "c", 11, 248},
+	// Test using multiple sequences to ensure they do not share state.
+	sequences := []Sequence{
+		NewSequence("bucket-name-1", "sequence-name-1"),
+		NewSequence("bucket-name-1", "sequence-name-2"),
+		NewSequence("bucket-name-2", "sequence-name-1"),
+		NewSequence("bucket-name-2", "sequence-name-2"),
 	}
-
-	for i, tc := range cases {
-		t.Run(fmt.Sprintf("case-%d", i), func(t *testing.T) {
-			s := NewSequence(tc.bucket, tc.name)
-			_, orig := s.curVal(db)
-
-			val := incrementN(s, db, tc.increments)
```

Comment:
I think it might still be cool to test increment method with various arguments.

Reply:
`incrementN` no longer exists.

`Sequence.increment` is not tested explicitly. You can trust that it works, because it is [used in tests to set up environment](https://github.com/iov-one/weave/blob/61039d6c9c8b8215a6e2127cba72f45ca0971658/orm/bucket_test.go#L152). If it was not increment the sequence correctly, the test would not succeed. Do you think this is enough, or it still requires explicit test?

Reply:
@husio we have 100% coverage there, so I guess this is fine